packages/discovery-provider/src/tasks/entity_manager/entities/comment.py

```diff
@@ -33,7 +39,7 @@ def create_comment(params: ManageEntityParameters):
         user_id=params.user_id,
         text=params.metadata["body"],
         entity_type=params.metadata["entity_type"],
-        entity_id=params.metadata["entity_id"],
+        entity_id=params.entity_id,
```

Comment:
interesting, so @DejayJD i think this might actually be incorrect though right, since this will be "Comment"?

Reply:
Yeah I think you're right, probably accidentally conflating the two (just like we got confused by yesterday lol)

Reply:
yea you're right it's confusing

Reply:
makes sense, I'll mess with it